src/fpti.js

```diff
@@ -56,7 +57,8 @@ export const FPTI_KEY = {
     PAY_NOW:                   ('pay_now' : 'pay_now'),
     STICKINESS_ID:             ('stickiness_id' : 'stickiness_id'),
     TIMESTAMP:                 ('t' : 't'),
-    OPTION_SELECTED:           ('optsel' : 'optsel')
+    OPTION_SELECTED:           ('optsel' : 'optsel'),
+    NATIVE:                    ('native' : 'native')
```

Comment:
Why do we need this if we already have `SDK_ENVIRONMENT`?

Reply:
@bluepnume I added this based on your comment here: https://github.com/paypal/paypal-smart-payment-buttons/pull/164#pullrequestreview-663588016

Reply:
Gotcha sorry I was talking about FPTI_STATE not FPTI_KEY. But it's fine, per discussion in other thread let's leave it out.

Reply:
@bluepnume got it, removed.